gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/kmeans/util/Point.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright © 2014 - 2021 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.kmeans.util;
+
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Point class that can be created by defining its coordinates.
```

Comment:
I know you got some ideas of an already existing code snippet. If you re-used code from other repositories, give a reference to it in the class comment. If not, everything is fine.

Reply:
Referenced https://github.com/TU-Berlin-DIMA/flink-tutorial.git repo